include/grpc/impl/codegen/slice.h

```diff
@@ -108,6 +108,10 @@ typedef struct {
   /** the number of slices allocated in the array. External users (i.e any code
    * outside grpc core) MUST NOT use this field */
   size_t capacity;
+  /** the index of the first slice who's memory is still owned by this buffer.
```

Comment:
I do not think this is needed. Note slice_buffer already has base_slices and slices to do essentially this. See below.

Reply:
Good point